packages-ts/gauntlet-cosmos-contracts/src/lib/constants.ts

```diff
@@ -43,8 +43,8 @@ export const TOKEN_UNIT = new BN(10).pow(new BN(TOKEN_DECIMALS))
 export const UATOM_DECIMALS = 6
 
 export const EMPTY_TRANSMITTERS = [
-  'cosmos1deadg6umpcwdr5ayj737vt49agnpxc6e5ct65t',
-  'cosmos1deadmlvdvtnnclqvdt526fz465w7v3vkd5r3h7',
-  'cosmos1deadz8q0v8yf4vw6jtnwfush2yka230jggws9p',
-  'cosmos1deadxqgqq63t9hxragwcqd2h020m2hvt3l49va',
+  'wasm1hft9sxhx7d7furw9y0rjxu4hfsm76ehkman78g',
```

Comment:
wasmd instead of cosmos bech32 prefix

Reply:
are these randomly generated wasm-prefixed addresses? I noticed the cosmos-prefixed ones start with `dead` to make it clear that they are burn addresses/empty transmitters - are we able to do something similar?

Reply:
oo good point. I don't know how they did that but will look into it

Reply:
Yeah these are just random addresses I generated with a script